src/components/search/dto/search-results.dto.ts

```diff
@@ -33,6 +30,7 @@ const publicSearchable = {
   Story,
   LiteracyMaterial,
   Song,
+  Location,
```

Comment:
Location is no longer an interface so it needs be removed below in `searchableAbstracts`